### ISSUE TITLE
Gluu-3.1.6-oxd-server fails to start after restarting VM

### ISSUE THREAD

**Issue by mogluu** [GluuFederation/oxd#311]
`oxd-server` installed while installing casa 3.1.6  does not start by itself after restarting the whole system.

```
service oxd-server restart
Stopping oxd-server: start-stop-daemon: warning: failed to kill 4017: No such process
1 pids were not killed
No process in pidfile '/var/run/oxd-server.pid' found running; none killed.
OK
Starting oxd-server:
PID: [2351]
OK Thu Apr 18 10:18:37 UTC 2019
```

**Comment by mogluu** [comment]
[oxd-server.zip](https://github.com/GluuFederation/oxd/files/3100039/oxd-server.zip)

**Comment by nikdavnik** [comment]
oxd-server fils to start after restart: This issues have been fixed in version 4.0

**Comment by yuriyz** [comment]
Closing since it's fixed in 4.0 as reported by @nikdavnik
